index.js

```diff
@@ -128,3 +133,18 @@ function byLocation(location) {
 function getCoercer(type) {
   return COERCION_STRATEGIES[type];
 }
+
+function pathsep(format) {
+  switch (format) {
+    case 'csv':
```

Comment:
The PR relates to the "collectionFormat" that defines the delimiter an array type parameter uses.  See [github.com/OAI/OpenAPI-Specification/blob/master/schemas/v2.0/schema.json#L1512](url) & [http://swagger.io/specification/#parameterObject](url)

Other swagger node modules coerce the parameter arrays from strings into actual arrays using the delimiter defined, or 'csv' as a default.